core/sdk-core/src/main/java/software/amazon/awssdk/core/SdkClient.java

```diff
@@ -32,4 +32,11 @@ public interface SdkClient extends SdkAutoCloseable {
      * @return name for this service.
      */
     String serviceName();
+
+    /**
+     * Service client configuration that contains region and ClientOverrideConfiguration
```

Comment:
Can we change this to be more general such as "The service client configuration exposes client settings to the user, for instance region and ClientOverrideConfiguration".

Reply:
updated